src/js/containers/aboutTheData/AgencyTableMapping.jsx

```diff
@@ -30,55 +30,101 @@ Tooltip.propTypes = {
     className: PropTypes.string
 };
 
+export const publicationsSubColumnPeriodFilters = {
+    2020: {
```

Comment:
Recommend adding a comment to clarify that these are the sub-columns that get _removed_ for each FY